istio-control/istio-discovery/templates/telemetryv2.yaml

```diff
@@ -0,0 +1,113 @@
+{{- if and .Values.pilot.telemetry.enabled .Values.pilot.telemetry.v2.enabled }}
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
```

Comment:
The authoritative source of this configuration will be in istio/istio e2e tests.
Can you restructure it so that the template "reads" from a local file. We can easily keep the local file in sync with the istio/istio

Reply:
I added two separate local files for metadata-exchange filter and stats filter